src/components/AnchorForCommentsOnly/BaseAnchorForCommentsOnly.js

```diff
@@ -48,6 +48,7 @@ const BaseAnchorForCommentsOnly = (props) => {
     return (
         <PressableWithSecondaryInteraction
             inline
+            style={[styles.cursorDefault, {fontSize: props.style.fontSize}]}
```

Comment:
not setting `display` to `inline` here as it will pass the style to the native component as well which doesn't support the `inline` value.